.github/workflows/precommit.yml

```diff
@@ -6,10 +6,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 11
+      # todo lower back to 11 after figuring out
+      # why precommit fails w/ 11 and lucene 9
+      - name: Set up JDK 14
         uses: actions/setup-java@v2
         with:
-          java-version: 11
```

Comment:
@nknize why `14`? I think we should ether keep JDK-11 or use JDK-17, JDK-14 is unsupported and long dead.

Reply:
Is it because of https://bugs.openjdk.java.net/browse/JDK-8209058, if not - could you point to the log please? thanks!